language/stdlib/transaction_scripts/testnet_mint.move

```diff
@@ -0,0 +1,13 @@
+script {
+use 0x1::LibraAccount;
+use 0x1::Signer;
+
+/// Send `amount` coins to `payee`.
+fun testnet_mint<Token>(payer: &signer, payee: address, amount: u64) {
```

Comment:
Should this enforce a max on `amount`?  I think for now, any `amount` smaller than the DD's balance will succeed.

Things will get somewhat better after we address https://github.com/libra/libra/issues/4277: any `amount` over the  TR threshold will fail. But even then we might want to fail with a nice error code here + say what the error code means in the script documentation.

Reply:
I'll add some enforcement in this script, and then once we add in TR for DDs we can remove :)